### DISCUSSION TITLE
different scaling policies for different triggers

### DISCUSSION THREAD

**Discussion by yuvalweber** [kedacore/keda#4216] [Q&A / Need Help]
I don't know if it's possible but I was wondering if there is an option to have different scaling policies for different triggers.
For example if I use the cron scaler and I want it to scale automatically to the number of replicas I mentioned, but if I just use a prometheus scaler I want it to scale 3 pods at the time.
Is it possible?

**Comment by zroubalik** [comment]
That is not possible currently. There are some feature requests opened for some more flexible configuration of the evaluation of scaling when there are multiple scalers.
